packages/lab/src/list-next/ListNext.css

```diff
@@ -0,0 +1,34 @@
+/* Default variables applied to the root element */
+.saltList {
+  --list-background: var(--salt-container-primary-background);
+  --list-borderStyle: var(--salt-container-borderStyle);
+  --list-borderWidth: var(--salt-size-border);
+  --list-height: calc(var(--salt-size-stackable) * var(--list-displayedItemCount));
+}
+
+/* Style applied to the root element */
+.saltList {
```

Comment:
I'd call this ListNext to ensure it doesn't get confused until the renames are done